### PR TITLE
APIMANAGER-5351: Upgrading swagger code-gen libraries for security vulnerability issues

### PR DESCRIPTION
**Pull request by abimarank**
JiRA : https://wso2.org/jira/browse/APIMANAGER-5351

### CODE REVIEW

File: swagger-parser/1.0.21.wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            io.swagger.parser.*;version="${export.pkg.version.swagger-parser}"
+                        </Export-Package>
+                        <Import-Package>
+                            !io.swagger.parser.*,
+                            org.slf4j; version="1.7.12",
```
Comment (Niranjan-K) [inline swagger-parser/1.0.21.wso2v1/pom.xml]: Need to define a range.

File: swagger-codegen/2.2.0.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    <url>http://wso2.org</url>
+
+    <distributionManagement>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+
+        <snapshotRepository>
```
Comment (thusithathilina) [inline swagger-codegen/2.2.0.wso2v1/pom.xml]: Please remove snapshot repo

File: swagger-codegen/2.2.0.wso2v1/pom.xml [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            TypeScript-node.*;version="${export.pkg.version.swagger-codegen}",
+                            TypeScript-Angular.*;version="${export.pkg.version.swagger-codegen}",
+                            tizen.*;version="${export.pkg.version.swagger-codegen}",
+                            swift.*;version="${export.pkg.version.swagger-codegen}",
+                            swagger-static.*;version="${export.pkg.version.swagger-codegen}",!io.swagger.models.*,
```
Comment (thusithathilina) [inline swagger-codegen/2.2.0.wso2v1/pom.xml]: Better to put ! pkgs first

File: swagger-parser/1.0.21.wso2v1/pom.xml [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    <url>http://wso2.org</url>
+
+    <distributionManagement>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+
+        <snapshotRepository>
```
Comment (thusithathilina) [inline swagger-parser/1.0.21.wso2v1/pom.xml]: Remove snapshot repo

File: swagger-parser/1.0.17.wso2v1/pom.xml [hunk 5]
```diff
@@ -111,6 +111,7 @@
 
     <properties>
         <export.pkg.version.swagger-parser>1.0.17.wso2v1</export.pkg.version.swagger-parser>
+        <slf4j.import.version>[1.7.0, 1.8.0)</slf4j.import.version>
```
Comment (thusithathilina) [inline swagger-parser/1.0.17.wso2v1/pom.xml]: This is a already released bundle. Can't alter released bundles

**Pull request merged**